documentation/asciidoc/computers/configuration/boot_folder.adoc

```diff
@@ -15,7 +15,7 @@ This is the bootloader, which is loaded by the SoC on boot, does some very basic
 ==== start.elf, start_x.elf, start_db.elf, start_cd.elf, start4.elf, start4x.elf, start4cd.elf, start4db.elf
 
 These are binary blobs (firmware) that are loaded on to the VideoCore in the SoC, which then take over the boot process.
-`start.elf` is the basic firmware, `start_x.elf` includes camera drivers and codec, `start_db.elf` is a debug version of the firmware, and `start_cd.elf` is a cut-down version with no support hardware blocks like codecs and 3D, and for use when `gpu_mem=16` is specified in `config.txt`. More information on how to use these can be found in xref:config_txt.adoc#boot-options[the `config.txt` section].
+`start.elf` is the basic firmware, `start_x.elf` includes camera drivers and codec, `start_db.elf` is a debug version of the firmware, and `start_cd.elf` is a cut-down version with no support hardware blocks such as cameras, codecs and 3D as well as initial framebuffer limitations, and is used when `gpu_mem=16` is specified in `config.txt`. More information on how to use these can be found in xref:config_txt.adoc#boot-options[the `config.txt` section].
```

Comment:
"with no support hardware blocks" should probably be "with no support for hardware blocks".
And in this brief overview, I'm not sure if it's worth mentioning "initial framebuffer limitations"? At the very least, it should probably be "as well as having initial framebuffer limitations". And given how long this sentence has become now, it should probably be split into two?

Reply:
Thanks, updated!